src/sidebar/services/annotations-exporter.ts

```diff
@@ -83,14 +83,20 @@ export class AnnotationsExporter {
     const annotationsText = annotations
       .map((annotation, index) => {
         const page = pageLabel(annotation);
+        const lines = [
+          annotation.created,
+          annotation.text,
+          extractUsername(annotation),
+          `"${quote(annotation)}"`,
+          annotation.tags.length > 0
+            ? `Tags: ${annotation.tags.join(', ')}`
+            : undefined,
+          page ? `Page: ${page}` : undefined,
+        ].filter(Boolean);
```

Comment:
I moved all lines to an array where we can conditionally filter out those that are undefined.

This will allow us to change the order of the lines without affecting the logic.